virt/virt-learn-more-about-openshift-virtualization.adoc

```diff
@@ -1,109 +1,66 @@
+* Install the xref:../virt/install/installing-virt-web.adoc#virt-installing-virt-operator_installing-virt-web[{VirtProductName} Operator] and the xref:../virt/install/virt-enabling-virtctl.adoc#virt-enabling-virtctl[`virtctl` tool].
+
+[id="virt-getting-started"]
+== Getting started
+
+Create a virtual machine:
+
+* Create a xref:../virt/virtual_machines/virt-create-vms.adoc#virt-creating-vm-wizard-web_virt-create-vms[RHEL virtual machine] using the wizard.
+* Create a Windows virtual machine:
+** Create and customize link:https://cloud.redhat.com/blog/virtual-machines-as-code-with-openshift-gitops-and-openshift-virtualization[Windows boot sources]
+** Create a xref:../virt/virtual_machines/virt-create-vms.adoc#virt-creating-vm-wizard-web_virt-create-vms[Windows virtual machine] using the wizard.
+** Install xref:../virt/virtual_machines/virt-installing-qemu-guest-agent.adoc#virt-installing-virtio-drivers-installing-windows_virt-installing-qemu-guest-agent[VirtIO drivers] on the virtual machine.
+
+Connect to a virtual machine:
+
+* Connect to a virtual machine using xref:../virt/virtual_machines/virt-accessing-vm-consoles.adoc#virt-connecting-to-the-terminal_virt-accessing-vm-consoles[the terminal] or xref:../virt/virtual_machines/virt-accessing-vm-consoles.adoc#virt-vm-serial-console-web_virt-accessing-vm-consoles[the serial console] in the {product-title} web console.
+* Connect to a virtual machine using SSH:
+** xref:../virt/virtual_machines/vm_networking/virt-using-the-default-pod-network-with-virt.adoc#virt-creating-a-service-from-a-virtual-machine_virt-using-the-default-pod-network-with-virt[Create a service] to expose a route for the virtual machine.
+** Use the service to connect to the virtual machine with xref:../virt/virtual_machines/virt-accessing-vm-consoles.adoc#virt-accessing-vmi-ssh_virt-accessing-vm-consoles[SSH].
+* xref:../virt/virtual_machines/virt-controlling-vm-states.adoc#virt-controlling-vm-states[Stop, start, pause, and restart] the virtual machine.
+
+[id="virt-getting-started-next-steps"]
+== Next steps
+
+Configure networks:
+
+* Configure the virtual machine for the xref:../virt/virtual_machines/vm_networking/virt-using-the-default-pod-network-with-virt.adoc#virt-using-the-default-pod-network-with-virt[default pod network].
+* Configure a Linux bridge network by creating a xref:../virt/virtual_machines/vm_networking/virt-attaching-vm-multiple-networks.adoc#virt-creating-linux-bridge-nncp_virt-attaching-vm-multiple-networks[creating a Linux bridge node network configuration policy], xref:../virt/virtual_machines/vm_networking/virt-attaching-vm-multiple-networks.adoc#virt-creating-linux-bridge-nad-web_virt-attaching-vm-multiple-networks[creating a network attachment definition], and xref:../virt/virtual_machines/vm_networking/virt-attaching-vm-multiple-networks.adoc#virt-vm-creating-nic-web_virt-attaching-vm-multiple-networks[attaching a virtual machine to the network].
```

Comment:
If this is going to be released only in 4.11, we should link to NMState operator and standalone NMState docs http://file.rdu.redhat.com/bgaydos/CNV-16812/networking/k8s_nmstate/k8s-nmstate-about-the-k8s-nmstate-operator.html

Reply:
I added a reference to this topic in the Linux Bridge bullet in Next Steps.